.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: rust
+rust: stable
+cache: cargo
```

Comment:
This tends to be problematic quite quickly as the cache never prunes and thus grows large. Travis syncs the cache via an S3 bucket download and thus it takes quite a while to sync e.g. 2GB. Cf. how we setup the caching strategy in dex-services or ethcontract-rs

Reply:
I was following the travis guide for Rust: https://docs.travis-ci.com/user/languages/rust/
Now I have read https://docs.travis-ci.com/user/caching/ where I noticed two things:

> To use the caching feature, in your repository settings, set Build pushed branches to ON.

I don't have this enabled so apparenlty the cache was not working anyway.

> Cache archives are currently set to expire after 28 days for repositories on https://travis-ci.org and 45 days for those on https://travis-ci.com. This means a specific cache archive will be deleted if it wasn’t changed after its expiration delay.

If I understand this correctly the cache should only grow for a limited number of days.

Reply:
I assume that as long as we run the CI once in 28 days it will slightly change the cached contents and just not stop growing after a limited number of days. 

You are right about the "official" docs, however we found that both in dex-services and ethcontracts the default caching was quickly becoming slower than not caching at all and followed [this guide](https://levans.fr/rust_travis_cache.html) to improve performance.